active/fellowship.md

```diff
@@ -2,30 +2,67 @@
+- **Django community experience**: WG members generally speaking should have been members of the Django community for some time. Once again this isn't a hard requirement, but one of the duties of the WG is to help Fellows navigate the community, if needed, so people with experience there are nice to have on the board.
+
+Membership is open to volunteers, subject to board approval. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest. Please include a brief explanation of how your background aligns with the goals above.
+
+Members serve a 1-year term, April 1st to March 31st, in alignment with the Fellow contracts. At the end of this term, they need to opt into staying involved to keep being
+a member of the group. If any member wishes to leave the group before the end of their term, they can do so without a vote.
 
 ## Budget
 
-The DSF Board sets the Fellowship compensation, but the WG then has the power to select who fills the position.
+- The Board sets the Fellowship compensation (which is adjusted yearly, automatically, to match CPI inflation).
+- The Board establishes a yearly Travel & Expenses budget, which Fellows may spend against (with management approval). This is set to a level to allow Fellows to attend at least one DjangoCon.
+  - Expenses beyond this yearly budget may (probably will!) be approved, but would require approval by the board to increase the budget.
 
 ## Comms
 
-TBD
+The Fellowship-WG has a private mailing list, and uses the DSF Slack. Fellows themselves generally work in the open, on the Django Forum and other public spaces.
 
 ## Reporting
 
-TBD
+The Fellowship WG is required to submit regular reports on the impact and efficacy of the Fellow program.
+
+Frequency: at least quarterly, with a strong preference for some sort of live "dashboard" (read: Google Sheet) that's updated more regularly for the Board to check any time.
+
+The specifics of what needs to be in here will change over time, and should be an ongoing conversation between Fellows, the Fellowship-WG, and the Board. It can and should change and evolve over time. It should probably should include things like count of tickets triaged and PRs reviewed, along with more qualitivave info like highlights of major features reviewed/shepherded, mentorship of new/existing contributors, talks at conferences, etc. Fellows already record much of this in their weekly reports to the community, so the WG's responsibility is to roll this up into longer-term metrics, highlight themes, and help make sure the information in these reports is useful to the Board and other consumers (sponsors, community members, etc).
```

Comment:
```suggestion
The specifics of what needs to be in here will change over time, and should be an ongoing conversation between Fellows, the Fellowship-WG, and the Board. It can and should change and evolve over time. It should probably include things like count of tickets triaged and PRs reviewed, along with more qualitative info like highlights of major features reviewed/shepherded, mentorship of new/existing contributors, talks at conferences, etc. Fellows already record much of this in their weekly reports to the community, so the WG's responsibility is to roll this up into longer-term metrics, highlight themes, and help make sure the information in these reports is useful to the Board and other consumers (sponsors, community members, etc).
```